ballista/rust/core/src/execution_plans/unresolved_shuffle.rs

```diff
@@ -117,4 +119,12 @@ impl ExecutionPlan for UnresolvedShuffleExec {
             }
         }
     }
+
+    fn statistics(&self) -> Statistics {
+        // We could try to fetch the statistics here from the shuffle writer,
```

Comment:
I think this comment is now out of date (as in this PR is actually doing what is suggested in the PR)

Reply:
Agreed, let me clarify this